test/ut/sdk/test_compressor_torch.py

```diff
@@ -152,7 +152,7 @@ def test_torch_slim_pruner(self):
         config_list = [{'sparsity': 0.2, 'op_types': ['BatchNorm2d']}]
         model.bn1.weight.data = torch.tensor(w).float()
         model.bn2.weight.data = torch.tensor(-w).float()
-        pruner = torch_pruner.SlimPruner(model, config_list)
+        pruner = torch_pruner.SlimPruner(model, config_list, optimizer=None, trainer=None, criterion=None)
```

Comment:
`optimizer` `trainer` `criterion` is necessary for SlimPruner and user needs to give three more `None` every time?